lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -31,7 +31,7 @@ def self.run(params)
         UI.user_error!("Couldn't find binary at path #{binary_path}") unless File.exist?(binary_path)
         binary_type = binary_type_from_path(binary_path)
 
-        auth_token = fetch_auth_token(
+        auth_token = params[:access_token] || fetch_auth_token(
```

Comment:
The `access_token` param does not exist yet. You'd have to add it to `self.available_options` below.

Reply:
Thanks!